styles/lichess/catppuccin.user.css

```diff
@@ -580,5 +622,43 @@
     .form-control:invalid {
       border-color: @red;
     }
+    #dasher_app .links a:hover, #dasher_app .links a:hover::before, #dasher_app .links button:hover, #dasher_app .subs .sub:hover {
+    background-color: @green;
+    color: @mantle;
+    }
+    #dasher_app .head::before {
+    color: @green;
+    }
+    /* Misc */
+    .daily-feed__update__marker.nobg {
+    background-color: @mantle;
+    border-color: @peach;
+    }
+    .daily-feed__updates {
+    border-color: @peach;
+    }
+    bad,.crosstable .loss {
+    color: @red
+    }
+    good, .crosstable .win, .user-link.online .line {
+    color: @green
+    }
+    #dasher_app .piece .no-square.active {
+    background-color: @green;
+    }
+    #dasher_app .selector button.active {
+    background-color: @green!important;
+    color: @mantle!important;
+    }   
+    #dasher_app .selector button.active::before {
+    color: @mantle!important;
+    }
+    #dasher_app .selector button:hover {
+    background-color: darken(@green, 30%);
+    color: @mantle;
+    }
+    :root:has(body[data-theme="system"]) #topnav.hover section:hover > a, :root:has(body[data-theme="system"]) #topnav section:active > a, :root:has(body[data-theme="system"]) #topnav div a, :root:has(body[data-theme="system"]) #dasher_app .selector button, :root:has(body[data-theme="system"]) #dasher_app .head, :root:has(body[data-theme="system"]) #dasher_app .piece .no-square, :root:has(body[data-theme="system"]) #dasher_app .theme .list button, :root:has(body[data-theme="system"]) #dasher_app .links a, :root:has(body[data-theme="system"]) #dasher_app .links button, :root:has(body[data-theme="system"]) #dasher_app .subs .sub, :root:has(body[data-theme="system"]) .site-buttons .dropdown a, :root:has(body[data-theme="system"]) .site-buttons .dropdown button {  
+     color: @surface1
+    }
```

Comment:
all the additional ` :root:has(body[data-theme="system"])` etc, seems a bit redundant.

Reply:
yea, shouldve removed it